code/game/area/Space_Station_13_areas.dm

```diff
@@ -2078,32 +2078,108 @@ var/list/teleportlocs = list()
+	/area/shuttle/ftl/crew_quarters/locker,
+	/area/shuttle/ftl/crew_quarters/locker/locker_toilet,
+	/area/shuttle/ftl/crew_quarters/fitness,
+	/area/shuttle/ftl/crew_quarters/cafeteria,
+	/area/shuttle/ftl/crew_quarters/kitchen,
+	/area/shuttle/ftl/crew_quarters/bar,
+	/area/shuttle/ftl/crew_quarters/theatre,
+	/area/shuttle/ftl/engine,
+	/area/shuttle/ftl/engine/engine_smes,
+	/area/shuttle/ftl/engine/engineering,
+	/area/shuttle/ftl/engine/break_room,
+	/area/shuttle/ftl/engine/tool_storage,
+	/area/shuttle/ftl/engine/chiefs_office,
+	/area/shuttle/ftl/engine/secure_construction,
+	/area/shuttle/ftl/engine/gravity_generator,
+	/area/shuttle/ftl/medical/medbay,
+	/area/shuttle/ftl/medical/medbay_lobby,
+	/area/shuttle/ftl/medical/patients_rooms,
+	/area/shuttle/ftl/medical/cmo,
+	/area/shuttle/ftl/medical/virology,
+	/area/shuttle/ftl/medical/morgue,
+	/area/shuttle/ftl/medical/chemistry,
+	/area/shuttle/ftl/medical/surgery,
+	/area/shuttle/ftl/medical/genetics,
+	/area/shuttle/ftl/medical/sleeper,
+	/area/shuttle/ftl/research/lab,
+	/area/shuttle/ftl/research/xenobiology,
+	/area/shuttle/ftl/research/storage,
+	/area/shuttle/ftl/research/mineral_storeroom,
+	/area/shuttle/ftl/research/test_area,
+	/area/shuttle/ftl/research/mixing,
+	/area/shuttle/ftl/research/misc_lab,
+	/area/shuttle/ftl/research/server,
+	/area/shuttle/ftl/research/explab,
+	/area/shuttle/ftl/research/division,
+	/area/shuttle/ftl/assembly/chargebay,
+	/area/shuttle/ftl/assembly/robotics,
+	/area/shuttle/ftl/janitor,
+	/area/shuttle/ftl/hydroponics,
+	/area/shuttle/ftl/munitions,
+	/area/shuttle/ftl/munitions/office,
+	/area/shuttle/ftl/munitions/loading,
+	/area/shuttle/ftl/munitions/cannon,
+	/area/shuttle/ftl/security/main,
+	/area/shuttle/ftl/security/brig,
+	/area/shuttle/ftl/security/prison,
+	/area/shuttle/ftl/security/processing,
+	/area/shuttle/ftl/security/warden,
+	/area/shuttle/ftl/security/armory,
+	/area/shuttle/ftl/security/hos,
+	/area/shuttle/ftl/security/detectives_office,
+	/area/shuttle/ftl/security/range,
+	/area/shuttle/ftl/security/transfer,
+	/area/shuttle/ftl/security/nuke_storage,
+	/area/shuttle/ftl/security/checkpoint,
+	/area/shuttle/ftl/security/checkpoint2,
+	/area/shuttle/ftl/security/checkpoint/supply,
+	/area/shuttle/ftl/security/checkpoint/engineering,
+	/area/shuttle/ftl/security/checkpoint/medical,
+	/area/shuttle/ftl/security/checkpoint/science,
+	/area/shuttle/ftl/security/vacantoffice,
+	/area/shuttle/ftl/security/vacantoffice2,
+	/area/shuttle/ftl/cargo,
+	/area/shuttle/ftl/cargo/office,
+	/area/shuttle/ftl/cargo/storage,
+	/area/shuttle/ftl/cargo/plasmastorage,
+	/area/shuttle/ftl/cargo/qm,
+	/area/shuttle/ftl/cargo/mining,
+	/area/shuttle/ftl/turret_protected,
+	/area/shuttle/ftl/turret_protected/ai_upload,
+	/area/shuttle/ftl/turret_protected/ai,
+	/area/shuttle/ftl/telecomms/computer,
+	/area/shuttle/ftl/telecomms/server,
+	/area/shuttle/ftl/atmos,
+	/area/shuttle/ftl/atmos/equipment,
+	/area/shuttle/ftl/storage/tech,

```

Comment:
There's no reason to include all the subtypes, just include all the top level types (except space)